README.md

```diff
@@ -1,128 +1,109 @@
-
-## Playwright with Test Runner
-
-If you are using Playwright Test Runner to run your Playwright tests, you can run all your playwright-test tests on BrowserStack as well. Follow the steps below to run the sample tests in this repository:
-
-1. Clone this repository using `git clone https://github.com/browserstack/playwright-browserstack.git` (if not already done).
-2. Go inside the directory playwright-test using `cd playwright-test`
-3. Install the dependencies using `npm install`
-4. Put in your credentials in the file `fixtures.js` in the caps part.
-5. If you are trying to run your own tests on BrowserStack, then you need to ensure that you have configured the `projects` correctly in `playwright.config.js` file.
-6. Run the sample test script using `npm test` which runs all the tests inside tests directory across 5 browsers in BrowserStack.
-7. If you want to test against localhost or other private websites, then run the sample test script using `npm run test:local` which runs all the tests inside tests directory across 5 browsers in BrowserStack.
-8. If you want to run your tests locally, you just need to configure the `projects` without name `@browserstack` in `playwright.config.js` file.
+## BrowserStack Inception
+Assignment Duration: 4 hours
 
-## Facing issues?
+1. Go to www.browserstack.com and log in with your demo account
+2. Start a Live session, as a part of the automation script
+3. Within the Live session, perform a Google search for “BrowserStack”
 
-If you are facing any issue with any of the above or any other issue in trying to run your Playwright tests on BrowserStack, you can [reach out to support](https://www.browserstack.com/contact#technical-support), select product as `Automate` and post your query there.
+## Example Inception Automate Build deep link
+
+https://automate.browserstack.com/dashboard/v2/builds/6e4c7f5027d7ca11de8196910edf5f1d55eaa5ee/sessions/3934b86b54b0068fd5dbf1886c8e9730d1449c7f
+
+
+```typescript
+
+const { test } = require('../fixtures');
+const { expect } = require('@playwright/test');
+const Tesseract = require('tesseract.js');
+
+// 1. Go to www.browserstack.com and log in with your demo account
+// 2. Start a Live session, as a part of the automation script
+// 3. Within the Live session, perform a Google search for “BrowserStack”
+
+// 1. Go to www.browserstack.com
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://www.browserstack.com/');
+});
+
+test.describe('Browserstack Inception Nicholas onboarding', () => {
+
+  test('Browserstack Inception Omnibus POC', async ({ page }) => {
+    // log in with your demo account
+    await page.waitForSelector('text=Sign in',{state:'visible'});
+    await page.locator('text=Sign in').first().click();
+    await page.waitForSelector('#user_email_login')
+    await page.fill('#user_email_login', "nicholas+demo@browserstack.com" );
+    await page.fill('#user_password', process.env.BROWSERSTACK_DEMO_PASSWORD );
```

Comment:
Write in README about updating this value

Reply:
now in README.md

add environment variables
```
export BROWSERSTACK_DEMO_EMAIL=nicholas+demo@browserstack.com
export BROWSERSTACK_DEMO_PASSWORD=yourSuperSecurePassword
```